From 28a133c6102d14c07aa8a035b35bba0400e09b68 Mon Sep 17 00:00:00 2001
From: Tal Yacobi <tyacobi@habana.ai>
Date: Wed, 04 Jun 2025 12:59:05 +0300
Subject: [PATCH] [SW-221571] gaudi3: mgmt: set status for qsfp_led

Explicitly set the status of qsfp_led to okay
in the mgmt dtsi.

Disable the qsfp_led in palad where it's not used.

Change-Id: If741bc995f0098f23738f645fcb641f37f0e952f
Signed-off-by: Tal Yacobi <tyacobi@habana.ai>
---

diff --git a/boards/hl/gaudi3/hl_gaudi3_mgmt_palad.dts b/boards/hl/gaudi3/hl_gaudi3_mgmt_palad.dts
index 3dd953e..c71cbf4 100644
--- a/boards/hl/gaudi3/hl_gaudi3_mgmt_palad.dts
+++ b/boards/hl/gaudi3/hl_gaudi3_mgmt_palad.dts
@@ -89,3 +89,7 @@
 &qsfp1 {
 	status = "disabled";
 };
+
+&qsfp_led {
+	status = "disabled";
+};
diff --git a/dts/arc/hl_gaudi3_mgmt.dtsi b/dts/arc/hl_gaudi3_mgmt.dtsi
index b77b02d..518cd8d 100644
--- a/dts/arc/hl_gaudi3_mgmt.dtsi
+++ b/dts/arc/hl_gaudi3_mgmt.dtsi
@@ -308,6 +308,7 @@
 		reg = <0x25>;
 		led-ctrl-reg = <0x0a>;
 		node_label = "HL338_LED_CPLD";
+		status = "okay";
 
 		led0: led_0 {
 			node_label = "QSFP1_LED_Y";
